CIP-0068/README.md

```diff
@@ -277,27 +283,25 @@ files_details =
   {
     ? name : bounded_bytes, ; UTF-8
     mediaType : bounded_bytes, ; UTF-8
-    src : bounded_bytes ; UTF-8
+    src : uri,
     ; ... Additional properties are allowed
   }
 
+; Only use `image/*` media types (for example `image/png`, `image/jpeg`, `image/svg+xml`, etc.)
 metadata = 
   {
     name : bounded_bytes, ; UTF-8
-    image : bounded_bytes, ; UTF-8
-    ? mediaType : bounded_bytes, ; UTF-8
```

Comment:
(potentially requiring version bump to 2 or 3)

Reply:
Agreed!

Reply:
ìmage` is not removed, and `mediaType` was optional, so I'm not sure of the benefit of bumping the version?

Reply:
It would not be erroneous because additional properties are allowed:

```
; ... Additional properties are allowed
```

Reply:
thanks @SmaugPool - I missed that! 🤓 @Ryun1 we can talk about any other reasons why we might need the version bump @ the CIP meeting today, but it seems like we are OK without one (cc @KtorZ).